common/djangoapps/student/tests/test_certificates.py

```diff
@@ -42,12 +42,10 @@ def test_display_verified_certificate(self, enrollment_mode):
     @override_settings(CERT_NAME_SHORT='Test_Certificate')
     @patch.dict('django.conf.settings.FEATURES', {'CERTIFICATES_HTML_VIEW': True})
     def test_linked_student_to_web_view_credential(self, enrollment_mode):
-
-        test_url = u'{url}?course={course_id}'.format(
-            url=reverse('cert_html_view'),
-            course_id=unicode(self.course.id)  # pylint: disable=no-member
-        )
-
+        test_url = u'{url}'.format(url=reverse('cert_html_view',
```

Comment:
Can we move this code into a common get_certificate_url() function that can be used in all of these locations?